ci/docker/python-wheel-manylinux-201x.dockerfile

```diff
@@ -62,12 +65,14 @@ ARG build_type=release
 ENV CMAKE_BUILD_TYPE=${build_type} \
     VCPKG_FORCE_SYSTEM_BINARIES=1 \
     VCPKG_OVERLAY_TRIPLETS=/arrow/ci/vcpkg \
-    VCPKG_DEFAULT_TRIPLET=x64-linux-static-${build_type} \
+    VCPKG_DEFAULT_TRIPLET=${arch_short_alias}-linux-static-${build_type} \
     VCPKG_FEATURE_FLAGS=-manifests
 
 # TODO(kszucs): factor out the package enumeration to a text file and reuse it
 # from the windows image and potentially in a future macos wheel build
 RUN vcpkg install --clean-after-build \
+        boost-build:${arch_short_alias}-linux && \
```

Comment:
Need to install the boost-build prior installing the boost packages, otherwise vcpkg will raise an error.

Reply:
Can you keep this restliction as a comment in this file?

Reply:
Added.